easybuild/easyconfigs/c/CPMD/CPMD-4.1-CrayIntel-2016.10-intel17.eb

```diff
@@ -3,6 +3,7 @@ easyblock = 'MakeCp'
 
 name = 'CPMD'
 version = '4.1'
+versionsuffix = ''
```

Comment:
The MKL version should replace the libsci version or they are supposed to co-exist?
If it will replace you can remove this empty versionsuffix variable, if they will co-exist I suggest you add something like `-MKL` to the version suffix